1-mandatory/3-reading-list.js

```diff
@@ -8,18 +8,70 @@ Create an array of objects, where each object describes a book and has propertie
 - The title (a string)
 - Author (a string)
 - and alreadyRead (a boolean indicating if you read it yet)
-Write a funciton that loops through the array of books. For each book, log the book title and book author like so: 
-"The Hobbit by J.R.R. Tolkien"
-You should write and log at least 5 books. 
-You should modify the tests so that they contain the values that correspond to your books.
-In this style of testing it is typical to write out as strings exactly what you expect your output to be, 
-without using any variables or any logic like loops, template strings or if statements.
-*/
+**/
 
-const books = [];
+const books = [
+  {
+    title: "The Hobbit",
+    author: "J.R.R. Tolkien",
+    alreadyRead: true,
+  },
+  {
+    title: "The Map of Salt and Stars",
+    author: "Jennifer Zeynab Joukhadar",
+    alreadyRead: false,
+  },
+  {
+    title: "Dietland",
+    author: "Sarai Walker",
+    alreadyRead: true,
+  },
+  {
+    title: "A Place for Us",
+    author: "Fatima Farheen Mirza",
+    alreadyRead: false,
+  },  
+  {
+    title: "The House of Impossible Beauties",
+    author: "Joseph Cassara",
+    alreadyRead: true,
+  },
+    
+];
+// Write a funciton that loops through the array of books. For each book, log the book title and book author like so: 
+// "The Hobbit by J.R.R. Tolkien"
+// You should write and log at least 5 books. 
+// You should modify the tests so that they contain the values that correspond to your books.
+// In this style of testing it is typical to write out as strings exactly what you expect your output to be, 
+// without using any variables or any logic like loops, template strings or if statements.
 
 // exercise 1
-function logBooks() {}
+function logBooks(books) {
+  books.forEach((book) => {
+    if(book.alreadyRead === true){
```

Comment:
Correct implementation, can `if(book.alreadyRead === true)` be simplified a bit further?
Fyi, if there is only one line of code within the if-block or else-block, you can also do
```
book.already ? 
  console.log(`You've already read ${book.title} by ${book.author}`) :
  console.log(`You still need to read ${book.title} by ${book.author}`);
```

You can try on your side :)